### PR TITLE
Address the issues with the redis API spec/examples found by new validation

### PR DESCRIPTION
**Pull request by TimLovellSmith**
### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [N/A] If applicable, the PR references the bug/issue that it fixes. - no bug for this issue yet
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [X] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

**Comment by TimLovellSmith** [comment]
(note: pending agreement on whether exposing acessKeys this way is the best approach)

File: arm-redis/2016-04-01/examples/RedisCacheForceReboot.json
Comment (vishrutshah) [inline arm-redis/2016-04-01/examples/RedisCacheForceReboot.json]: Do you really need this (`Message`) to be PascalCase?

File: arm-redis/2016-04-01/swagger/redis.json
Comment (vishrutshah) [inline arm-redis/2016-04-01/swagger/redis.json]: Empty description.

File: arm-redis/2016-04-01/swagger/redis.json
Comment (vishrutshah) [inline arm-redis/2016-04-01/swagger/redis.json]: Do you really need this (`Message`) to be PascalCase?

**Review by vishrutshah — CHANGES_REQUESTED** [review]
@TimLovellSmith I've reviewed the changes & left some minor comments. 

There are some other errors, I've discovered apart from the changes you have made using AutoRest Linter but I am just going to list here for your reference. We'd strongly recommend fixing them as well for better SDK experiences. 

```
WARNING: NonAppJsonTypeWarning - Media types other than 'application/json' has limited support
        Path: #/Consumes[2]
WARNING: NonAppJsonTypeWarning - Media types other than 'application/json' has limited support
        Path: #/Produces[2]
WARNING: UniqueResourcePaths - More than one resource path is not allowed in a single spec
        Path: #/Paths
WARNING: LongRunningResponseValidation - An operation with x-ms-long-running-operation extension must have a valid terminal success status code. 200 or 201 for Put/Patch. 200, 201 or 204 for Post. 200 or 204 or both for Delete.
        Path: #/Paths/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/import/post/Extensions/x-ms-long-running-operation
WARNING: LongRunningResponseValidation - An operation with x-ms-long-running-operation extension must have a valid terminal success status code. 200 or 201 for Put/Patch. 200, 201 or 204 for Post. 200 or 204 or both for Delete.
        Path: #/Paths/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/export/post/Extensions/x-ms-long-running-operation
```

Secondly, based on email thread regarding `accessKeys` I am going to wait on approval of this PR until we have talk/decision with @hovsepm.

Thanks!

File: arm-redis/2016-04-01/examples/RedisCacheForceReboot.json
Comment (TimLovellSmith) [inline arm-redis/2016-04-01/examples/RedisCacheForceReboot.json]: I added this in the example due to an unfortunate fact that it *is* returning pascal case on the wire today.
We did not formally document this member before the exercise with examples. In fact when we reviewed this REST API we did not document any response body.

Comment (TimLovellSmith) [inline arm-redis/2016-04-01/examples/RedisCacheForceReboot.json]: I added this in the example due to an unfortunate fact that it *is* returning pascal case on the wire today.
We did not formally document this member before the exercise with examples. In fact when we reviewed this REST API we did not document any response body. Perhaps the best thing to do here is document the API as returning an empty response, and treat the fact that RP returns a Message property as a bug?

Comment (TimLovellSmith) [inline arm-redis/2016-04-01/examples/RedisCacheForceReboot.json]: @vishrutshah my take, revised: it should never have been PascalCase but now that it is, it can't be fixed without it being breaking, so its stuck that way til next api-version.

Comment (vishrutshah) [inline arm-redis/2016-04-01/examples/RedisCacheForceReboot.json]: @TimLovellSmith I understand your concerns and makes sense to wait until next api-version. It's be great if you could please open and issue on that here and from your side as well, we do not miss this on next api-version.

**Comment by TimLovellSmith** [comment]
@vishrutshah any further changes required?

File: arm-redis/2016-04-01/examples/RedisCacheForceReboot.json
Comment (TimLovellSmith) [inline arm-redis/2016-04-01/examples/RedisCacheForceReboot.json]: Opened #935

**Comment by vishrutshah** [comment]
@TimLovellSmith Changes looks good to me. 

@amarzavery If you have time to skim through, please skim or I can merge it by tomorrow :)

**Review by vishrutshah — APPROVED** [review]

**Comment by TimLovellSmith** [comment]
ping @amarzavery @vishrutshah Ready to take this one?

**Comment by amarzavery** [comment]
looking at this PR right now

**Review by amarzavery — APPROVED** [review]

**Comment by amarzavery** [comment]
LGTM

**Pull request merged**
